pkg/storage/access_token_file.go

```diff
@@ -1,25 +1,70 @@
 package storage
 
-import "github.com/cyberark/conjur-authn-k8s-client/pkg/storage/config"
+import (
+	"fmt"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/storage/config"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
 
 type AccessTokenFile struct {
-	TokenFilePath	string
-	Data	[]byte
+	TokenFilePath string
+	Data          []byte
 }
 
 func NewAccessTokenFile(config config.Config) (token *AccessTokenFile, err error) {
```

Comment:
We sure that this is a good name for this function? To me, `NewAccessTokenFile` represents a file and not the access token object that holds the data and the path. Can't we just leave it as NewAccessToken or do we have two AccessToken structs (1 for old flow with file path and 1 for new flow with token in memory?